backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/schema/create/Organization.kt

```diff
@@ -0,0 +1,27 @@
+package app.ehrenamtskarte.backend.application.webservice.schema.create
+
+import app.ehrenamtskarte.backend.application.webservice.schema.view.JsonField
+import app.ehrenamtskarte.backend.application.webservice.schema.view.Type
+import app.ehrenamtskarte.backend.application.webservice.utils.JsonFieldSerializable
+import com.expediagroup.graphql.annotations.GraphQLDescription
+
+data class Organization(
+    val name: String,
+    val address: String,
```

Comment:
Is the address the street, houseNumber, postalCode and location combined? Is there a reason why it's combined here but not in `PersonalData`?

Reply:
I think this was derived from some formulas in several communal districts.

Otherwise I don't so see any reason why we shouldn't structure it the same way for convenience.

Reply:
Same. I think we should just expand it as in PersonalData. Maybe make it a new data class `Address`? 🤔

Reply:
Sounds good to wrap the common parts in an Address class.